pkg/iostreams/color_windows.go

```diff
@@ -0,0 +1,15 @@
+package iostreams
+
+import (
+	"os"
+)
+
+func Is256ColorSupported() bool {
+	// Windows advertises neither TERM nor COLORTERM, but we assume that 256 colors are supported.
+	return true
+}
+
+func IsTrueColorSupported() bool {
+	// Windows Terminal supports true color.
+	return os.Getenv("WT_SESSION") != ""
```

Comment:
This isn't Windows-specific. Windows Terminal defines this environment variable in WSL2 as well, despite `TERM` being defined as `xterm-256color`. I think either how you had it is fine (other Windows terminals may define `TERM`, and hopefully WT will as well per the link I posted), or just add a check for `WT_SESSION`:

![image](https://user-images.githubusercontent.com/1532486/129965640-23a84069-d830-4bfd-bef5-c1a032feb5f7.png)